libraries/AP_HAL_ChibiOS/RCOutput.cpp

```diff
@@ -1329,9 +1329,6 @@ bool RCOutput::get_output_mode_banner(char banner_msg[], uint8_t banner_msg_len)
  */
 void RCOutput::cork(void)
 {
-    if (corked) {
```

Comment:
i'd remove this internal error, but not the other one
calling cork(),cork(),push() isn't great, but is not actually dangerous. Calling push() without cork() is the key issue

Reply:
Fixed